### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by legit-programmer**

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -18,6 +18,6 @@ export default async function Index() {
             </div>
         </div>
     ) : (
-        <h1 className="text-3xl font-bold">No Upcoming Competetions</h1>
+        <h1 className="text-2xl font-bold">No Upcoming Competetions</h1>
```
Comment (metaloozee) [inline app/page.tsx:21]: - [ ] keep the text's property similar to above (upcoming competitions) 
- [ ] change competetions to competitions

Comment (legit-programmer) [reply app/page.tsx]: fixed

File: app/page.tsx [hunk 2]
```diff
@@ -18,6 +18,8 @@ export default async function Index() {
             </div>
         </div>
     ) : (
-        <h1 className="text-2xl font-bold">No Upcoming Competetions</h1>
+        <h1 className="text-center text-2xl md:text-3xl font-light">
```
Comment (metaloozee) [inline app/page.tsx:21]: fix the spelling

File: app/page.tsx
Comment (metaloozee) [inline app/page.tsx:1]: fix the spelling in "Ongoing Competetions"

Comment (legit-programmer) [reply app/page.tsx:1]: alr done

Comment (legit-programmer) [reply app/page.tsx:1]: b03707785adec271c2a691db6bb966050623874f

**Review by metaloozee — APPROVED** [review]

**Pull request merged**